src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const allPopulationText = document.querySelectorAll('.population');
+const allPopulationNumber = [...allPopulationText].map(item => (
```

Comment:
- don't need to create a copy of allPopulationText as method map did it already 
```suggestion
const allPopulationNumber = allPopulationText.map(item => (
```
- item it's not a good name, try to provide more descriptive name

Reply:
it`s dont work. If doing like you write, it will be a collection, not an array (it haven`t map method)
![Снимок экрана 2023-11-17 160247](https://github.com/mate-academy/js_get_data_DOM/assets/82728301/203f0495-2d9f-4694-8370-481dc41138f6)